gradle/gradle-mvn-mpp-push.gradle

```diff
@@ -28,8 +64,10 @@ def getRepositoryPassword() {
 task emptySourcesJar(type: Jar) {
   classifier = 'sources'
 }
-task emptyJavadocJar(type: Jar) {
+
+task javadocsJar(type: Jar, dependsOn: dokka) {
   classifier = 'javadoc'
+  from dokka.outputDirectory
```

Comment:
Does this choose the right doc artifact for the associated platform or is there a single unified javadoc for all platforms?